src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -186,6 +186,20 @@ private static bool TryGetTextForOperator(SyntaxToken token, Document document,
                 return true;
             }
 
+            if (token.IsKind(SyntaxKind.InterpolatedStringStartToken) ||
+                token.IsKind(SyntaxKind.InterpolatedStringEndToken) ||
+                token.IsKind(SyntaxKind.InterpolatedStringTextToken))
+            {
+                text = "$_CSharpKeyword";
+                return true;
+            }
+
+            if (token.IsKind(SyntaxKind.StringLiteralToken) && token.ValueText[0] == '@')
+            {
+                text = "@_CSharpKeyword";
+                return true;
+            }
```

Comment:
Currently, the dotnet/docs doesn't have any `@$_CSharpKeyword` F1 keyword. I opened a PR to add this in docs. Are you okay with this new F1 keyword and adding the following here?

```csharp
        if (token.IsKind(SyntaxKind.InterpolatedVerbatimStringStartToken))
        {
            text = "@$_CSharpKeyword";
            return true;
        }
```

Reply:
@davidwengier Should I add the combination?

Reply:
Looks like that would regress the experience from a help page (albeit for string, which isn't great) to a generic 404 page, so I'd say leave it until after the doc page is created.

Reply:
@davidwengier I've a PR there
https://github.com/dotnet/docs/pull/20197